content/doc/tutorials/tutorial-for-installing-jenkins-on-IBM-Cloud.adoc

```diff
@@ -5,202 +5,230 @@ section: doc
+
+== Deploying Jenkins
+
+We will deploy Jenkins on our cluster.
+
+. Select *Catalog* in the upper left corner.
+. Select *Software* from the catalog.
+. Search for *Jenkins* and select it.
++
+image::tutorials/IBMCloud/search.png[IBM workspace search]
+
+.  Select *IBM Kubernetes Service*.
++
+image::tutorials/IBMCloud/target-select.png[IBM Kubernetes selection]
+
+. On the application page, select the _empty dot_ next to the cluster you wish to use.
++
+image::tutorials/IBMCloud/cluster-select.png[IBM cluster selection]
+
+. Select *Target namespace* and then choose the default Namespace or use a custom one.
++
+image::tutorials/IBMCloud/details-namespace.png[IBM space name]
+
+. Give a unique *name* to workspace, which you can easily recognize.
++
+image::tutorials/IBMCloud/details-names.png[IBM workspace name]
+
+. Select which resource group you want to use.
+This is for access control and billing purposes.
+For more information please visit link:https://cloud.ibm.com/docs/account?topic=account-account_setup#bp_resourcegroups[resource groups].
++
+image::tutorials/IBMCloud/details-resource.png[Resource details]
+
+. Give *tags* to your Jenkins, for more information visit link:https://cloud.ibm.com/docs/account?topic=account-tag[tags].
++
+image::tutorials/IBMCloud/details-tags.png[Default value parameters]
+
+. Select *Parameters with default values*.
+You can set custom deployment values or use the default ones.
++
+image::tutorials/IBMCloud/parameters.png[IBM parameters setup]
+
+. Set the Jenkins password in the parameters.
++
+image::tutorials/IBMCloud/password.png[create password instruction]
+
+. After finishing everything, *select* the box next to the agreements and select *install*.
++
+image::tutorials/IBMCloud/install.png[Installation instructions]
+
+. The Jenkins workspace will start installing, wait a couple of minutes for this to complete.
++
+image::tutorials/IBMCloud/in-progress.png[workspace installation screen]
+
+. Your Jenkins workspace has been successfully deployed.
++
+image::tutorials/IBMCloud/done.png[IBM workspace deployment screen]
+
+== Verifying the Jenkins installation
+
+. Go to link:http://cloud.ibm.com/resources[Resources] in your browser.
+. Select *Clusters*.
+. Select your cluster.
++
+image::tutorials/IBMCloud/resource-select.png[Resource selection screen]
+
+. Now you are at you clusters overview.
+. Select *Actions* in the top right corner of the page.
+. Select *Web terminal* in the dropdown menu.
++
+image::tutorials/IBMCloud/cluster-main.png[Clusters overview page]
+
+. Select *Install* and then wait for a couple of minutes.
++
+image::tutorials/IBMCloud/terminal-install.jpg[terminal install screen]
+
+. Once installation has completed, you will repeat this process.
+
+... Select *Actions*.
+... Select *Web terminal* and a terminal window will open.
+
+. *Type* in the terminal, be sure to change NAMESPACE to the namespace you choose at the deployment setup:
```

Comment:
Long-term, I think this sentence could be revised further, to provide more information for each code sample below. However, I'm not sure exactly what each sample is doing. So at a minimum, we could revise it to something like this?
```suggestion
. Type in the terminal. You must change `NAMESPACE` to the namespace you chose during the deployment setup
```